WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -557,6 +558,11 @@ def my_competitions
         :competitor_limit_reason,
         :remarks,
         :registration_requirements,
+        :on_the_spot_registration,
```

Comment:
I left this boolean because I think this is valuable information to store. Let me know if you think it should go too.

Reply:
I may be missing something, but I don't see what the valuable information this lets us store is. If there is no fee for on the spot registration, then the fee is 0, right?

It's very possible I'm missing something, so please feel free to explain to me what I'm missing.

Reply:
There are three possibilities:
1. No OTS registration.
2. OTS registration and NO fee. 
3. OTS registration and fee. 

Without the boolean, we won't be able to distinguish between 1. and 2. However, if you still think this is not important information to store, I can get rid of it.

Reply:
Oh GOSH, I'm dumb. Yeah, you're totally right. There's having OTS registration, and then there's how much it costs. Your database schema seems totally ok here.